api/kessel/inventory/v1beta1/k8s_cluster.proto

```diff
@@ -15,16 +13,11 @@ option java_outer_classname = "K8sClusterProto";
 
 message K8sCluster {
   // Metadata about this resource
-  Metadata metadata = 1 [ (google.api.field_behavior) = REQUIRED ];
-
-  // Write only reporter specific data
-  ReporterData reporter_data = 245278792
-      [ (google.api.field_behavior) = INPUT_ONLY ];
-
-  // The entities that registered this item in the Kessel Asset Inventory. The
-  // same resource may be registered by multiple reporters.
-  repeated ReporterData reporters = 353323086
-      [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  Metadata metadata = 1;
 
+  // Reporter specific data
+  ReporterData reporter_data = 245278792;
```

Comment:
Not related to this PR, but curious were the tag Number comes from? Is it a random number?